### DISCUSSION TITLE
How to disable extension

### DISCUSSION THREAD

**Discussion by triangkas** [shivammathur/homebrew-php#200] [Q&A / Help] [answered]
Hi, 

I have one question. I installed PHP 5.6 on Mac via shivammathur/php/php@5.6 and I want to disable extension soap.
I marked ";" on extension = php_soap.dll in file php.ini, but it doesn't work.

Thank you in advance.

**Comment by shivammathur** [comment, marked as answer]
@triangkas SOAP is not loaded as a shared extension, it is compiled into the php binary so it is not possible to disable it.

**Reply by triangkas** [reply]
Ok, Thank you very much for your information.
